lib/6to5/transformation/templates/umd-runner-body.js

```diff
@@ -2,6 +2,6 @@
   if (typeof define === "function" && define.amd) {
     define(AMD_ARGUMENTS, factory);
   } else if (typeof exports !== "undefined") {
```

Comment:
@maxnordlund Good point.
@webpro After that's changed made it should be good to merge.